docs/data/toolpad/core/api/components/dashboard-layout.md

```diff
@@ -0,0 +1,3 @@
+# Dashboard Layout API
```

Comment:
For the API, let's port the docs generator from core?

Reply:
We can do that in this separate PR, right? https://github.com/mui/mui-toolpad/pull/3536

Reply:
Ah, the X one, I was checking the Base one, will take a look.